Portfolio mobile.html

```diff
@@ -7,18 +7,196 @@
     <link rel="stylesheet" href="./styles.css"/>
 </head>
 <body>
-    <div class="Mobile-container">
-        <img id="top_left" src="./images/Ellipse%2018.png" alt="ellipse18"/>
-        <img id="top_right" src="./images/small-circle.png" alt="small-circle"/>
+    <!-- /*Desktop version*/ -->
+    <div class="desktop-container">
+        <img id="top_left" src="./images/circle%201%20header.png" alt="circle header 1"/>
+        <img id="top_right" src="./images/circle%202%20header.png" alt="circle header 2"/>
         <header class="top">
-        <a href="#">John Doe</a>
+            <a href="#">Shuhad Loofer</a>
+            <nav>
+                <a href="#">Profile</a>
+                <a href="#">About</a>
+                <a href="#">Contact</a>
+                <a href="#"><img src="./images/Union.svg" alt="Email"/></a>
+                </nav>
+        </header>
+        <section class="head">
+            <h2 class="main_head">Shuhad Loofer</h2>
+            <h2 class="support_head">Software Developer</h2>
+            <p class="main_text">I can help you build a product, 
+            feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, 
+            don't hestiate to contact me.
+            </p>
+            <div class="divider"><img src="./images/Vector%204.png" alt="divider"/></div>
+            <div class="title_head"> 
+                <ul>
+                    <li class="list">Language</li>
+                    <li>Javascript</li>
+                    <li>Ruby</li>
+                    <li>Html</li>
+                    <li>Css</li>
+                </ul>
+                <ul>
+                    <li class="list">Frameworks</li>
+                    <li>Bootstrap</li>
+                    <li>Ruby on Rails</li>
+                    <li>RSPec</li>
+                    <li>CapyBara</li>
+                    <li>Selenium</li>
+                </ul>
+                <ul>
+                    <li class="list">Skills</li>
+                    <li>Codekit</li>
+                    <li>Github</li>
+                    <li>Codepen</li>
+                    <li>Gitlab</li>
+                    <li>Terminal</li>
+                </ul>
+            </div>
```

Comment:
- [ ] Please consider adding Work and arrows to your design so it's almost like the provided one

|Your design|Figma design|
|-|-|
|![Screenshot_20220601_211103](https://user-images.githubusercontent.com/25227022/171495168-a60ee4b4-146f-48d0-bbac-f7ae9a4dee81.png)|![Screenshot_20220601_211237](https://user-images.githubusercontent.com/25227022/171495207-60d835b3-6dc9-439a-b0bb-39e2217c0825.png)|